client/package.json

```diff
@@ -6,14 +6,32 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "@types/dashify": "^1.0.0",
     "@types/jest": "^24.0.0",
     "@types/node": "^12.0.0",
+    "@types/object-hash": "^1.3.1",
+    "@types/pluralize": "^0.0.29",
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
+    "@types/react-modal": "^3.10.5",
+    "@types/react-router-dom": "^5.1.5",
+    "@types/styled-components": "^5.1.0",
+    "@votingworks/ballot-encoder": "^1.2.0",
+    "@votingworks/qrcode.react": "1.0.1",
+    "dashify": "^2.0.0",
+    "js-file-download": "^0.4.12",
+    "normalize.css": "^8.0.1",
+    "object-hash": "^2.0.3",
+    "pagedjs": "^0.1.40",
+    "pluralize": "^8.0.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-modal": "^3.11.2",
+    "react-router-dom": "^5.1.2",
     "react-scripts": "3.4.1",
-    "typescript": "~3.7.2"
+    "styled-components": "^5.1.0",
+    "typescript": "~3.7.2",
+    "zip-stream": "^3.0.1"
```

Comment:
This is used for creating zip files on the client. I don't think we need it or its associated `.d.ts` file?

Reply:
Yes, will remove. This is the downside of attempting to move fast by cargo copying from `election-manager`.